helix-core/src/main/java/org/apache/helix/tools/ClusterSetup.java

```diff
@@ -329,8 +329,9 @@ public HelixAdmin getClusterManagementTool() {
     return _admin;
   }
 
-  public void addStateModelDef(String clusterName, String stateModelDef, StateModelDefinition record) {
```

Comment:
Please do not remove public methods. Instead, call the new method from the old one with a default value.